internal/model/plan.go

```diff
@@ -0,0 +1,93 @@
+package model
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+// Plans define the structure for an Api Plan.
+// swagger:model
+type Plans struct {
+	//The id for the plan
+	// in: path
+	//required: true
+	ID string `gorm:"column:id;primaryKey,unique;type:uuid"`
+	// The Name for the plan
+	//required: true
+	Name string `gorm:"column:name;type:varchar(100)"`
+	// the Description for the plan
+	//required: true
+	Description string `gorm:"column:description;type:text"`
```

Comment:
A Plan has many PlanQuotaDefaults, so PlanQuotaDefaults should be referenced here as well. See [this documentation](https://gorm.io/docs/has_many.html) for details.

I don't think that anyone on the team is very familiar with GORM, so what we might want to do is to have a few of us get together to work through a few examples. It might be nice to be able to play with this to see if we can get it to do what we want.